src/Microsoft.SqlTools.ServiceLayer/LanguageServices/PeekDefinitionScripts.tt

```diff
@@ -32,7 +34,7 @@ namespace Microsoft.SqlTools.ServiceLayer.LanguageServices
 			///
 			var indent = "    ";
 			var directory = Path.GetDirectoryName(Host.TemplateFile);
-			string xmlFile = Path.Combine(directory, "PeekDefinitionSupportedTypes.xml");
```

Comment:
>PeekDefinitionSupportedTypes.xml [](start = 45, length = 32)

should we delete PeekDefinitionSupportedTypes.xml if we're not using it anymore?